solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -575,6 +662,10 @@ describe("RandomBeacon - Relay", () => {
           // relayEntryHardTimeout`.
           await mineBlocks(64 * 10 + 5760)
 
+          // Simulate group creation is already in progress.
+          await (randomBeacon as RandomBeaconStub).publicDkgLockState()
+          await (randomBeacon as RandomBeaconStub).publicCreateGroup()
```

Comment:
Why do we need to call these here? `publicCreateGroup` seems not to be needed and we are calling `publicDkgLockState` in the context for "and DKG is awaiting seed" at line 700.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2691/commits/166a6c8d38f900aa6d0e2df4c72243d870c3e25a